tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -763,6 +777,9 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args):
                 args['input_model'] = decoder
                 args["framework"] = "pytorch"
 
+        if not inp_model_is_object:
```

Comment:
can we hide this condition inside `update_args_for_unnamed_saved_model` function.

`unnamed` looks interesting. Any other ideas?:)

Reply:
`update_args_for_saved_model_dir`?

Also, instead of `inp_model_is_object`, can we use a check `isinstance(..., (str, Path))` inside the function?

Reply:
Yes, sure

Reply:
Done.